Plugins~/Src/MeshSyncClientBlender/msblenModifiers.cpp

```diff
@@ -0,0 +1,330 @@
+#include "msblenModifiers.h"
+#include <sstream>
+#include <MeshSync/SceneGraph/msMesh.h>
+#include "MeshSync/SceneGraph/msPropertyInfo.h"
+#include <msblenUtils.h>
+#include "BlenderPyObjects/BlenderPyContext.h"
+#include "msblenBinder.h"
+
+namespace blender {
+#if BLENDER_VERSION < 300
+void msblenModifiers::exportProperties(const Object* obj, ms::PropertyManager* propertyManager) {}
+void msblenModifiers::importProperties(std::vector<ms::PropertyInfo> props) {}
+#else
+
+// Copied from blender source that we cannot include:
```

Comment:
Because we are not compiling all of Blender source files.
We are only using Blender's include files, and some of its implementation files.
And for this to work, we hand picked the implementation code that we need and put them here.
A bit messy, yes. 
I tried to include all Blender source files before but it wasn't as easy as I thought.


FYI: @schinkowski 
This change is actually taken from another PR, and has been merged to dev
That is why I think we should do rebase before doing a code review, or otherwise it would show up like this occasionally.

Reply:
I apologize. It seems that I had a different file in mind when commenting on this yesterday.